Model/Notification.php

```diff
@@ -359,6 +360,69 @@ public function setDone($done)
         return $this->setData(self::DONE, $done);
     }
 
+    /**
+     * Gets the Processing flag for the notification.
+     *
+     * @return bool Processing.
+     */
+    public function getProcessing()
+    {
+        return $this->getData(self::PROCESSING);
+    }
+
+    /**
+     * Sets Processing flag.
+     *
+     * @param bool $processing
+     * @return $this
+     */
+    public function setProcessing($processing)
+    {
+        return $this->setData(self::PROCESSING, $processing);
+    }
+
```

Comment:
These getter and setter for processing are not needed for this PR but I noticed that they were missing